stable/airflow/requirements.lock

```diff
@@ -1,9 +1,9 @@
 dependencies:
 - name: postgresql
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.13.1
+  version: 8.1.4
```

Comment:
Can you remove this file, their is no reason to commit the lock file actually !

Reply:
I see! Sure